packages/documentation/src/stories/misc/migration-guide/migrationv9-10.component.ts

```diff
@@ -107,24 +106,80 @@ export class MigrationV99Component extends LitElement {
 
       <section>
         <ol class="bubble-tea">
+          <li>
+            <h3>Introduction</h3>
+            <p>
+              Version 10 comes with a <span class="fw-bold">new look and cleaner codebase</span> —
+              and yes, a few breaking changes, all for good reason.
+            </p>
+            <p>
+              We’ve completely refreshed the design and reworked how components are built.
+              <span class="fw-bold">Bootstrap</span> and
+              <span class="fw-bold">Ng-Bootstrap</span> are gone; everything now runs on
+              <span class="fw-bold">Stencil Web Components</span>, which means the Design System
+              works across <span class="fw-bold">any framework</span> (<a
+                href="/?path=/docs/833ef689-a573-40f5-a6a6-30a999b94733--docs"
+                >Angular</a
+              >, <a href="/?path=/docs/13b9c7f1-993d-4348-a3b7-a7ceb92fd5c7--docs">React</a>, or
+              <a href="/?path=/docs/edfb619b-fda1-4570-bf25-20830303d483--docs">plain HTML</a>).
+            </p>
+            <p>
+              Most utility classes are still there, but renamed to be
```

Comment:
Reading "Most utility classes are still there" makes me less certain about the update.

```suggestion
              We've reworked utility classes to be 
```